app/presenters/menu/default_menu.rb

```diff
@@ -89,6 +89,7 @@ def physical_infrastructure_menu_section
         Menu::Section.new(:phy, N_("Physical Infrastructure"), 'fa fa-plus fa-2x', [
           Menu::Item.new('physical_infra_overview', N_('Overview'),  'physical_infra_overview', {:feature => 'physical_infra_overview'},               '/physical_infra_overview'),
           Menu::Item.new('ems_physical_infra',      N_('Providers'), 'ems_physical_infra',      {:feature => 'ems_physical_infra_show_list'},          '/ems_physical_infra'),
+          Menu::Item.new('physical_chassis',        N_('Chassis'),   'physical_chassis',        {:feature => 'physical_chassis_show_list'}, '/physical_chassis'),
```

Comment:
Just a minor thing: could you please align `'/physical_chassis'` with the lines above? (whitespaces fix). Thanks.